pkg/imagestore/imagestore_test.go

```diff
@@ -544,13 +544,15 @@ var _ = Describe("HaveVersion", func() {
 	It("is true for versions that are present", func() {
 		Expect(store.HaveVersion("4.8", "x86_64")).To(BeTrue())
 		Expect(store.HaveVersion("4.9", "arm64")).To(BeTrue())
+		Expect(store.HaveVersion("4.15", "s390x")).To(BeTrue())
```

Comment:
This test will fail until you update the versions variable in the test block.

Reply:
Thx. Added the versions in the test block.